README.md

```diff
@@ -147,7 +147,7 @@ const baseStyle = {
 
 export default {
   components: {
-    SvelteInVue: vueAdaptor(SvelteApp, baseStyle, "div")
+    SvelteInVue: toVue(SvelteApp, baseStyle, "div")
```

Comment:
The same imported adapter as `toVue` but was trying to use `vueAdapter`.
Fixed this too.